lib/initializers/watch-for-disconnected.ls

```diff
@@ -0,0 +1,15 @@
+intercept = require 'intercept-stdout'
+
+module.exports = (robot) !->
+  timer = null
+
+  unhook_intercept = intercept (txt) !->
+
+    if txt.match //slack client closed, waiting for reconnect//i
+
+      clearTimeout timer
+      timer := setTimeout (-> throw new Error 'Force restarting due to disconnect'), 60 * 1000
```

Comment:
Need to extract 60 * 1000 to some global to make the tests less brittle.  Also consider pulling the reconnect and disconnect strings to global constants.